.github/workflows/build-angular.yml

```
@@ -0,0 +1,125 @@
+name: Build Angular
```

```suggestion
name: build-angular
```
match the name with the file name